### ISSUE TITLE
Banner and Interstitial will not show for iOS

### ISSUE THREAD

**Issue by emerwang** [floatinghotpot/cordova-admob-pro#501]
Hi, I am checking out the demo and it will not show any banner or interstitial in ios simulator. Also it seems like the buttons for the simple demo do not work for ios (it is not really responsive to click)

I used the following command for demo.
 plugin-verify cordova-plugin-admobpro

Android works fine. And the rest of the example like games and ionic work fine. 

I have also tried using the admob_simple.js set up with my own admob ad unit id for banner and interstitial. But no ad seems to appear. Would really appreciate some help here!

**Comment by floatinghotpot** [comment]
I just verified the plugin `plugin-verify cordova-plugin-admobpro ios`, it displayed the banner and interstitial correctly.

Which country are you located in?

**Comment by emerwang** [comment]
I am in the US. I don't think location is the issue here since I can receive ads in android and the ionic app. Also using the lastest Xcode and cordova-plugin-admobpro.  I will try to see if the showbanner function is being triggered or not. Meanwhile, is there anything you would recommend? Maybe running in an actual device? Reinstall the plugin?

**Comment by floatinghotpot** [comment]
One possible reason of failure is using an ad unit id created for android platform in an iOS app.

Please be aware that, you need create separate ad unit ids for different platforms.

In admob portal, 
1) create an app for android, then create an ad unit for this app.
2) create an app for ios, then create an ad unit for this app.

**Comment by emerwang** [comment]
Hi floatinghotpot:

Here are some updates from my side. I think what concerned me most is that even the plugin-verify demo will not display any banner or interstitial for ios. The buttons are simply not responding and no add is being served.

Here are some details of my environment:
macOS Sierra 10.12.3
cordova 6.5.0
Simulator iphone SE 10.2

I think it is highly likely this is a bug with iOS 10.  I have tried doing the same thing with my old laptop that runs iOS 9.3 and the same demo app runs perfectly (with buttons responding and banner ads showing correctly). Is the plugin currently supporting ios 10.2?

I also did some debug with a project that included the admob_simple.js. It seems like the code after AdMob variable never executes in admob_simple. I think this could be due to AdMob var being undefined and the entire code crashes without getting to the line:
 if (! AdMob ) { alert( 'admob plugin not ready' ); return; }
